torch/onnx/utils.py

```diff
@@ -467,6 +468,7 @@ def _model_to_graph(model, args, verbose=False,
 
     params_dict = _get_named_param_dict(graph, params)
 
+    all_input_and_param_names = [val.debugName() for val in graph.inputs()]
```

Comment:
Is "all_input_and_param_names" used in the code?

Reply:
Good catch. Forgot to remove it.